pkg/cmd/open-svc.go

```diff
@@ -108,6 +109,7 @@ func NewCmdOpenService(streams genericclioptions.IOStreams) *cobra.Command {
 	}
 
 	cmd.Flags().IntVarP(&o.port, "port", "p", o.port, "The port on which to run the proxy. Set to 0 to pick a random port.")
+	cmd.Flags().StringVar(&o.portName, "port-name", o.portName, "The service port name. default is empty and uses the first port")
```

Comment:
The output of the `kubectl get svc` command does not include the port name, so I would like to support port numbers with the same flag.

```
kube-prometheus-stack               kube-prometheus-stack-alertmanager               ClusterIP   10.100.7.172     <none>        9093/TCP,8080/TCP              22d
```


Therefore, the flag name will be changed to `svc-port` and I would like to be able to support both the name and the number.

Port number support is not needed in this PR.

```suggestion
	cmd.Flags().StringVar(&o.svcPort, "svc-port", o.svcPort, "The service port name. default is empty and uses the first port")
```

Reply:
> Therefore, the flag name will be changed to `svc-port` and I would like to be able to support both the name and the number.

I agree! Thanks for the advice.

I support service port number separately.